build/resolver/git/entrypoint.sh

```diff
@@ -144,7 +141,22 @@ wrapPull() {
         failed=$(mkdir $PULLING_LOCK > /dev/null 2>&1 || echo fail)
         if [[ $failed != "fail" ]]; then
             echo "Got the lock, start to pulling..."
-            pull
+            if ! pull; then
```

Comment:
This error judgment should be fine-tuned, only *bitbucket* server git clone with a *certain* depth-caused error will be retried.